components/os/windows.go

```diff
@@ -81,6 +82,14 @@ func getWindowsRepositoryURL(version AgentVersion) string {
 }
 
 func getAgentURL(version AgentVersion) (string, error) {
+	// TODO: using the testing repository on Windows will require more work
+	// - there is no installers_v2.json file to use
+	// - the pipeline ID is not enough to find the exact filename to use, as it is structured like:
+	// datadog-agent-<last tag>.git.<number of commits since tag>.<commit short sha>.pipeline.<pipeline id>-1-x86_64.msi
+	if version.Repository == TestingRepository {
```

Comment:
Yes - the logic is slightly more complex than on Linux (because the paths for channels are not consistent across our repos - see `getWindowsRepositoryURL`), but the same channel values work on Linux and Windows.